test/example.css

```diff
@@ -2,6 +2,10 @@ html {
     box-sizing: border-box;
 }
 
+a {
+    animation: 100ms;
```

Comment:
move this property down into one of the larger blocks below so we're also demonstrating property order.